esx_service/utils/kvESX.py

```diff
@@ -22,6 +22,10 @@
         c_void_p, c_char_p, c_int32, c_bool, c_uint32, c_uint64
 import json
 import logging
+import sys
+
+# Python version 3.5.1 
+PYTHON64_VERSION = 50659824
```

Comment:
Ok, I'm checking if the Python version is >= 3.5.1. This is lowest version
thats reporting the issues and what 6.5 is going to be shipping with
(main-stage has this).

On Tue, Jun 28, 2016 at 3:10 AM, Mark Sterin <notifications@github.com>
wrote:

> In esx_service/utils/kvESX.py
> <https://github.com/vmware/docker-volume-vsphere/pull/479#discussion_r68661402>
> :
>
> > @@ -22,6 +22,10 @@
> >          c_void_p, c_char_p, c_int32, c_bool, c_uint32, c_uint64
> >  import json
> >  import logging
> > +import sys
> > +
> > +# Python version 3.5.1
> > +PYTHON64_VERSION = 50659824
>
> if python clicks to 3.5.2, will it still work ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/vmware/docker-volume-vsphere/pull/479/files/618f13068273be9c4251cba120e77e704c061ed4#r68661402>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/APHseGJMhfvP8cxbzLRI3MRfrQ9zh3_wks5qQEM2gaJpZM4I4qrt>
> .
>